### DISCUSSION TITLE
User input for ww3_prep

### DISCUSSION THREAD

**Discussion by fed1942** [NOAA-EMC/WW3#898] [Q&A] [answered]
Hello, my name is Federico Haspert and I am a beginner in the use of WW3. Can someone give me a simple example of the format a wind input file must have in order to be read by ww3_prep? An example for the smallest possible grid (3x3) would be ideal.

Thank you in advance.

**Comment by aliabdolali** [comment, marked as answer]
Hi,
Here is one one example, for current but it is the same for wind, if you change the variable name from CUR to WND.

regtests/ww3_tp1.6/input/ww3_prep_curr.inp

You can search regtests/input directories and find other examples.

Another way would be to run WW3 with homogeneous wind and then use ww3_outf to generate ascii field output. The format is the same as the format of an input ascii file.

**Reply by fed1942** [reply]
Thak you for the answer. I was able to see the file and I have another question: what happens with the direction of the wind?

**Reply by aliabdolali** [reply]
You should provide uwnd and vwnd not windspeed and wind direction.
For current velocity the same, ucur an vcur are provided. 
If you have 3*3 input grid, you will have 2*3*3 values
